src/asound/20-bluealsa.conf

```diff
@@ -32,7 +32,7 @@ ctl.bluealsa {
 }
 
 pcm.bluealsa {
-	@args [ DEV PROFILE DELAY SRV ]
+	@args [ DEV PROFILE VOL CODEC DELAY SRV ]
```

Comment:
Since the order of arguments determines order of values without arg name, I would put `codec` before `vol`. I think that in most cases where PCM is given on command line the codec information will be more common, e.g.:
`bluealsa:33:22:33:22:11:33,a2dp,aac` rather than `bluealsa:33:22:33:22:11:33,a2dp,70`. The initial volume will be more likely in the custom asoundrc file, IMHO.
Also, one more thing. Maybe add codec names in the header file `shared/dbus-client.h` and add helper function for case insensitive mappings between the value for codec argument and actual codec name. The codec in BlueALSA D-Bus API is case sensitive, but for users it will be more convenient to use codec names as case insensitive. Also, I would add aliases for common variants, e.g. `aptx` and `apt-x`.